documentation/licenses/msmpi/license.tex

```diff
@@ -0,0 +1,32 @@
+\documentclass[a4paper, 12pt]{article}
+
+\usepackage{enumitem}
+
+\begin{document}
+
+\title{MIT License}
```

Comment:
this needs to be added in the  licenses_folders table dynawo_algorithms_documentation.sh, compile.sh and clean.sh

Reply:
Done